src/main/webapp/app/shared/pipes/artemis-duration.pipe.ts

```diff
@@ -1,24 +1,24 @@
 import { Pipe, PipeTransform } from '@angular/core';
-import * as moment from 'moment';
+import dayjs from 'dayjs';
 
 @Pipe({ name: 'durationTo' })
 export class DurationPipe implements PipeTransform {
     /**
      * Calculate the duration of an event from the diff of two dates.
-     * @param startDate The start date time of the event. Must be convertible to moment().
-     * @param endDate The end date time of the event. Must be convertible to moment().
+     * @param startDate The start date time of the event. Must be convertible to dayjs().
+     * @param endDate The end date time of the event. Must be convertible to dayjs().
      * @returns string 'HH:mm'
      */
-    transform(startDate: Date | moment.Moment | string | number | null | undefined, endDate: Date | moment.Moment | string | number | null | undefined): string {
-        if (!endDate || !moment(endDate).isValid() || !startDate || !moment(startDate).isValid()) {
+    transform(startDate: Date | dayjs.Dayjs | string | number | null | undefined, endDate: Date | dayjs.Dayjs | string | number | null | undefined): string {
```

Comment:
I added this decision to the confluence page, as I say that we currently have both version in the code base, we can decide on one version and change it in a refactoring PR later on.

Reply:
why should we exclude null?  if for some reason the startDate is null (I know that is would be againt the coding guidelines), this would crash. We not just leaving it, as it is?

Reply:
Shouldn't this be covered by the TypeScript type check?

I generally understand your point but I dislike just checking for it anyway because this makes the guidelines a joke since the reason such guideline exists is exactly to not have to do additional checks everywhere.